src/Controller/FeedController.php

```diff
@@ -31,10 +32,13 @@ public function index(DogCRUDService $dogService, PostRepository $postRepository
         
         $posts = $postRepository->findAll();
 
+        $comments = $commentRepository->findAll();
+
```

Comment:
Do you want all the comments or just the comments for the posts you fetch? Also, do you really want to get all the posts here too?

We can get the logged in user in the framework, so we should probably get the posts for the dog's friends. There are repository methods for this now, so we should probably use them